### ISSUE TITLE
[Enhancement] Does it possible to make option for sound... 

### ISSUE THREAD

**Issue by WhiteMagicRaven** [NovaSquirrel/Mesen-X#131]
Just look that great video, explaining on how sound can be enhanced highly
https://www.youtube.com/watch?v=8RrQrATnXXY&t=68s&ab_channel=KYLXBN

**Comment by retro-64K** [comment]
yeah, mainly with the vrc6, i'm pretty sure it's possible, though some work is obviously required.
